### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34163]
#### VFS team name
Debt Resolution

#### VFS product name
Medical Copays (MCP)

#### Point of Contact/Reviewers
Noah Gelman (@noahgelman) - Accessibility

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
The value of an ARIA ID must be unique to prevent other instances from being overlooked by assistive technologies.Failing Elements My VAMy HealthProfileSign Out
 <div class="va-dropdown-panel " id="account-menu" hidden=""> - Sarah Koomson

### Link, screenshot or steps to recreate
https://googlechrome.github.io/lighthouse/viewer/

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
In order to prevent invalid markup, a unique ID must be implemented, so JAWS/NVDA does not overlook the second instance. Could possibly use - "id="MYVA-account-menu", "id="MyHealth-account-menu", etc/

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by craigwheeler** [comment]
hey @noahgelman I saw you were listed as a point of contact on this ticket so wanted to reach out to verify what the issue is here? The ticket states there's an issue with some `aria id` but isn't too clear on where the issue was reported or what should be updated? I've also verified any aria tags used are unique so maybe not an issue anymore?

**Comment by shiragoodman** [comment]
@craigwheeler  - Noah is out sick today but I'll be sure he see's this when he returns. Did you see the Platform Recommendation section above? There's additional details on the issue.

**Comment by craigwheeler** [comment]
hey @shiragoodman thanks! yep I saw the platform recommendation but the suggestion is to add `id="MYVA-account-menu` (add to what i dunno?) and the only instance of `account-menu` I can find relates to the sign in profile menu which is where i'm confused on what should be updated with MCP?

**Comment by shiragoodman** [comment]
@craigwheeler The Staging Review for Medical Copays is tomorrow at 11:30am ET so feel free to ask for clarification then. If Noah is out sick still, we have a couple other accessibility specialists that will be in attendance :)

**Comment by craigwheeler** [comment]
spoke with the accessibility team during staging review and verified this is an issue with the header and was asked to leave the ticket in backlog for them to address it at a later time

**Comment by craigwheeler** [comment]
it appears that the issue highlighted for this ticket occurs in the sign in profile widget for `account-menu` found here:

![Screen Shot 2022-01-27 at 9.29.46 AM.png](https://images.zenhubusercontent.com/5f355b2a6fe9a1b76ba980d4/4145b9ea-a8d5-4fff-9720-3e405e829ed9)

**Comment by shiragoodman** [comment]
@denisecoveyduc @brianalloyd I have removed the vsa-debt team label and added the public websites label. Brian and team are going to investigate the issue and determine a solution. If the issue ends up being the responsibility of another team, please let me know and I will get it reassigned. Thanks both!

**Comment by sara-amanda** [comment]
Two tickets relating to this issue noted in comments of the respective tickets. Details in #54863 @briandeconinck @joshkimux @coforma-terry

**Comment by laflannery** [comment]
I cannot replicate this any more. We are also in the process of rebuilding the header so I am going to make sure there aren't any duplicate IDs in the new implementation. I'm going to close this due to those reasons but if anyone feels like it should be reopened please let me know
